common/beerocks/scripts/prplmesh_utils.sh.in

```diff
@@ -287,21 +287,46 @@ status_function() {
     pgrep -l ieee1905_transport
     pgrep -l local_bus
 
-    # check for operational status
-    LOGS_PATH=@BEEROCKS_LOG_FILES_PATH@
+    bridge_mac="$(ip link show dev br-lan | awk '/^ *link\/ether / {print $2}')"
```

Comment:
Bridge name is not always br-lan, it can be brlan0 or anything else

Reply:
This is why it fails on RDKB

The bridge name needs to be fetched with `@BEEROCKS_BRIDGE_IFACE@`

Reply:
Fixed